selenium_webdriver/src/test/resources/environments/dev.properties

```diff
@@ -0,0 +1,15 @@
+#===============Test_browsers_configuration================
+sauceDemo.login.page.url=https://www.saucedemo.com/
+sauceDemo.main.page.url=https://www.saucedemo.com/inventory.html
+standard.user.name=standard_user
+standard.user.password=secret_sauce
+locked.user.name=locked_out_user
+locked.user.password=secret_sauce
+problem.user.name=problem_user
+problem.user.password=sauce_demo
+performance.glitch.user.name=performance_glitch_user
+performance.glitch.user.password=sauce.demo
+unsuccessful.login.message=Epic sadface: Sorry, this user has been locked out.
```

Comment:
I'm not sure it's a property just an expected result. Please remove

Reply:
fixed